scripts/installation/install.sh

```diff
@@ -78,7 +78,91 @@ delete_crds() {
   kubectl delete -f build/crds --timeout=90s
 }
 
+create_kb(){
+cd ${SEDNA_ROOT}
+kubectl create namespace sedna
+kubectl label node/$KB_NODE_NAME sedna=kb --overwrite
```

Comment:
this will overwrite the gm node, if KB_NODE_NAME is same with GM_NODE_NAME

Reply:
set label: sedna=controller for gm and kb